sljit_src/sljitNativeARM_64.c

```diff
@@ -80,8 +80,21 @@ static const sljit_u8 freg_map[SLJIT_NUMBER_OF_FLOAT_REGISTERS + 3] = {
 #define BLR 0xd63f0000
 #define BR 0xd61f0000
 #define BRK 0xd4200000
+#define CAS 0xc8a07c00
+#define CASA 0xc8e07c00
+#define CASAB 0x08e07c00
+#define CASAH 0x48e07c00
+#define CASAL 0xc8e0fc00
+#define CASALB 0x08e0fc00
+#define CASALH 0x48e0fc00
+#define CASB 0x08a07c00
+#define CASH 0x48a07c00
+#define CASL 0xc8a0fc00
+#define CASLB 0x08a0fc00
+#define CASLH 0x48a0fc00
```

Comment:
Is it worth to add instruction which we don;t use?

Reply:
FWIW I used them while developing, so had been not only encoded but verified to be correct and could save some time when we have a need to use them later.